cmd/ddev/cmd/import-files.go

```diff
@@ -108,7 +111,8 @@ func promptForExtPath(val *string) {
 }
 
 func init() {
-	ImportFileCmd.Flags().StringVarP(&sourcePath, "src", "", "", "Provide the path to the source directory or tar/tar.gz/tgz/zip archive of files to import")
-	ImportFileCmd.Flags().StringVarP(&extPath, "extract-path", "", "", "If provided asset is an archive, optionally provide the path to extract within the archive.")
+	ImportFileCmd.Flags().StringVarP(&target, "target", "t", "", "Target upload dir, defaults to the first upload dir")
+	ImportFileCmd.Flags().StringVar(&sourcePath, "src", "", "Provide the path to the source directory or tar/tar.gz/tgz/zip archive of files to import")
```

Comment:
But hasn't this changed to --file? Or how does that work? I thought we changed from --src to --file everywhere.

Reply:
No, so far I changed the db-* commands only. Here it's about a source directory or an archive but `file` is the wrong term for a directory and that's why I've left it out in the other patch. But could be changed to `source` with shorthand `s`.

Reply:
Change like suggested